modules/light-clients/08-wasm/types/upgrade_test.go

```diff
@@ -101,7 +103,7 @@ func (suite *TypesTestSuite) TestVerifyClientMessage() {
 			clientState := endpoint.GetClientState()
 
 			clientMsg = &types.ClientMessage{
-				Data: []byte{1},
```

Comment:
I could have left this as it was, but I like using tests as some sort of documentation as well, and thus being explicit that we expect a header here it's more helpful for the reader, I think.